bitnami/harbor/templates/chartmuseum/chartmuseum-svc.yaml

```diff
@@ -13,9 +14,9 @@ metadata:
   {{- end }}
 spec:
   ports:
-    - port: {{ include "harbor.chartmuseum.servicePort" . }}
-      name: http
-      targetPort: http
+    - port: {{ ternary 443 80 .Values.internalTLS.enabled }}
```

Comment:
Don't we want to allow custom port names in the service? We've seen examples where, because of firewall issues, the port number had to be forced.

Reply:
All these microservices are exposed via ClusterIP services and that's why I'm hardcoding the ports or reusing the same values used for the "container ports". I'm opened to make it customizable. As you prefer.

Reply:
I'd allow customizations even though we set ClusterIP by default. We've done it in most of the charts except for the headless services.

Reply:
I'd allow customizations for the services ports (they'll be independent parameters)